faker/tests/__init__.py

```diff
@@ -601,6 +601,31 @@ def test_ipv6(self):
                 re.compile(r'^([0-9a-f]{0,4}:){2,7}[0-9a-f]{0,4}/\d{1,3}$').search(
                     address))
 
+    def test_lorem_text(self):
+        from faker.providers import lorem, BaseProvider
+        from faker.providers.lorem import el_GR, la, ru_RU
+
+        text_provider = eval("lorem." + lorem.default_locale + ".Provider(None)")
+        int_provider = BaseProvider(None)
+
+        for _ in range(999):
+            min_nb = int_provider.random_int(min=5, max=200)
+            max_nb = min_nb + int_provider.random_int(max=200)
+            lorem_text = text_provider.text(min_nb_chars=min_nb, max_nb_chars=max_nb)
+            self.assertTrue(max_nb >= len(lorem_text) >= min_nb)
+            self.assertTrue(lorem_text[-1] == '.')
+            self.assertFalse(re.findall(r'\.\s[A-Z][a-z]*[\.]*[\s]*$', lorem_text))
+            self.assertFalse(lorem_text[-2] == ' ')
+            self.assertTrue(all(map(lambda x: x[0].isupper(), lorem_text.split('. '))))
```

Comment:
those tests are a little obscure. Can you add some comments clarifying what it's checking?